src/gleam_stdlib.mjs

```diff
@@ -564,6 +564,8 @@ export function classify_dynamic(data) {
     return "Map";
   } else if (typeof data === "number") {
     return "Float";
+  } else if (data == null) {
+    return "Nil";
```

Comment:
This should be `"null"` as `Nil` is `undefined` rather than `null`.

Reply:
See there I was just plain wrong. Bad gut!

Reply:
We both were ^^

I'll change it to "Null" once I'm back at a computer.

I'll also change it to strict equality again, otherwise it would report that "Null" was passed in when "Nil" was passed in. 
I'll also add a check for "Nil" specifically.

Where I'm not sure is: Should I add a type called "Null" that maps to "null" or does that already exist?

Reply:
Done, except for adding a new gleam type called `Null` for the JS type `null`.

Reply:
The string isn't the name of any specific type, rather it's more vaguely what you could call the value. There's no specific `Null` type in Gleam

Reply:
> You've also added a case for Nil. That's cool, but could you add a test for the classify functin with that value too?

I've added deno tests for the `decode_map` function that cover that.

> We'll need to update the Erlang implementation also.

Done...I think. Not sure if you meant changing "Atom" to "Nil" or adding a case for "Nil" and adding "Atom". I don't know Erlang 😬

Reply:
`nil` specifically should be `Nil`, the rest should be `Atom`. it'll be something like this:

```erlang
classify(nil) -> <<"Nil">>;
classify(X) when is_atom(X) -> <<"Atom">>;
...
```

Reply:
done abd0c9113c0b4f029fd86122d1885ce2b2590a01